services/minespace-web/common/utils/Validate.js

```diff
@@ -108,6 +108,11 @@ export const exactLength = memoize((min) => (value) =>
 export const number = (value) =>
   value && Number.isNaN(Number(value)) ? "Input must be a number" : undefined;
 
+export const positiveNumber = (value) =>
```

Comment:
Would be awesome if we could add a couple of tests for this one, the other validations have tests defined in `src/tests/utils/Validate.spec.js`

Reply:
A test added for `positiveNumber`